changelogmanager/changelog.py

```diff
@@ -116,13 +116,13 @@ def release(self, override_version: Optional[str] = None) -> None:
         if str(_version) in self.get().keys():
             raise logging.Error(
                 file_path=self.get_file_path(),
-                message=f"Unable release already released version '{_version}'",
+                message=f"Unable to release an already released version '{_version}'",
```

Comment:
Is it worth mentioning this grammar fix in CHANGELOG.md?

Reply:
I wouldn't to be honest